src/ResourceManager/PowerBIDedicated/Commands.PowerBIDedicated/Commands/SuspendAzureRmPowerBIEmbeddedCapacity.cs

```diff
@@ -0,0 +1,91 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Azure.Commands.PowerBIDedicated.Models;
+using Microsoft.Azure.Management.PowerBIDedicated;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.PowerBIDedicated.Properties;
+using Microsoft.Azure.Commands.PowerBIDedicated.Utilities;
+using Microsoft.Azure.Management.PowerBIDedicated.Models;
+
+namespace Microsoft.Azure.Commands.PowerBIDedicated
+{
+    [Cmdlet(VerbsLifecycle.Suspend, "AzureRmPowerBIEmbeddedCapacity",
+        SupportsShouldProcess = true),
+        OutputType(typeof(AzurePowerBIDedicated))]
+    public class SuspendAzurePowerBIEmbeddedCapacity : PowerBIDedicatedCmdletBase
+    {
+        [Parameter(Position = 1, ValueFromPipelineByPropertyName = true,
+            Mandatory = false, HelpMessage = "Name of resource group under which to retrieve the capacity.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = true,
```

Comment:
@selasagi please remove the `ValueFromPipelineByPropertyName` property from `ResourceGroupName` and `Name`

Reply:
Done